helm/alfresco-content-services/7.3.N_values.yaml

```diff
@@ -32,6 +32,14 @@ share:
 activemq:
   image:
     tag:
+alfresco-transform-service:
+  transformrouter: *ats-router
+  pdfrenderer: *ats-pdf
+  imagemagick: *ats-im
+  libreoffice: *ats-lo
+  tika: *ats-tika
+  transformmisc: *ats-misc
+  filestore: *ats-fs
```

Comment:
why not applying the same approach of values.yaml also here?

Reply:
I just forgot about it.